cmd/operator-sdk/up/local.go

```diff
@@ -167,6 +167,10 @@ func upLocalHelm() error {
 }
 
 func setupOperatorEnv() error {
+	// Set local env
+	if err := os.Setenv(k8sutil.ForceRunModeEnv, "local"); err != nil {
```

Comment:
`OSDK_FORCE_RUN_MODE` isn't used by Ansible or Helm runners, so there's no point in setting it here.

Reply:
I stand corrected. This variable is checked by callee's of `Run()` for both Ansible and Helm.

@joelanford @fabianvf will the runners treat a local operator differently in non-Go cases?